web/pandas/pdeps/0014-translate-website-content.md

```diff
@@ -5,137 +5,65 @@
 - Discussion: [#56301](https://github.com/pandas-dev/pandas/issues/56301)
               [#57204](https://github.com/pandas-dev/pandas/pull/57204)
 - Author: [Albert Steppi](https://github.com/steppi),
-- Revision: 1
+- Revision: 2
 
 ## Abstract
 
 The suggestion is to have official translations made for content of the core
-project website [pandas.pydata.org](https://pandas.pydata.org) and provide a
-language drop-down selector on [pandas.pydata.org](https://pandas.pydata.org)
-similar to what currently exists at [numpy.org](https://numpy.org).
+project website [pandas.pydata.org](https://pandas.pydata.org) and offer
+a low friction way for users to access these translations on the core
+project website.
 
+## Motivation, Scope, Usage, and Impact
 
-## Motivation and Scope
+There are many potential users with no or a low level of English proficiency
+who could benefit from quality official translations of the Pandas website
+content. Though translations for all documentation would be valuable,
+producing and maintaining translations for such a large and oft-changing
+collection of text would take an immense and sustained effort which may
+be infeasible. The suggestion is instead to have translations made for only
+a key set of pages from the core project website.
```

Comment:
I am correct in thinking that anyone would be able to open a PR on the mirror site (if this would be the solution).

documentation changes are often good first issues.

I assume its the approval process that "would take an immense and sustained effort which may be infeasible."

Otherwise do we have an off ramp for when the translator funding ends?

Reply:
Yes, anyone could open a PR, and anyone could contribute translations on Crowdin after asking for an invite. Our hope is to set up a compounding snowball type effect, where we can help build a community of volunteer translators who can help keep translations up to date.

Reply:
Great. pandas is foremost a community driven volunteer project so this aligns well with the project values.